test/testcases.py

```diff
@@ -23,6 +23,12 @@ class TestCase:
     # rootfs to use (e.g. ext4), some containers like fedora do not
     # have a default rootfs. If unset the container default is used.
     rootfs: str = ""
+    # osinfo_template is a string template describing the OS detected by
+    # 'osinfo-detect'. It can contain '{arch}' that will be replaced with the
+    # actual container image arch
+    osinfo_template: str = ""
+    # Sign the container_ref and use the new signed image instead of the original one
+    sign: bool = False
```

Comment:
This will conflict with https://github.com/osbuild/bootc-image-builder/pull/680 (see details why osinfo_template is not ideal as part of the testcase class) - having sign here is good though.